WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/ProductRowViewModelTests.swift

```diff
@@ -426,10 +426,193 @@ final class ProductRowViewModelTests: XCTestCase {
+                                          subscription: subs)
+
+        // When
+        let viewModel = ProductRowViewModel(id: rowID, product: product, productSubscriptionDetails: subs)
+
+        // Then
+        XCTAssertEqual(viewModel.subscriptionBillingDetailsLabel, "$5.00 / 1 month")
+    }
+
+    func test_subscriptionBillingDetailsLabel_when_periodInterval_is_more_than_1_then_returns_pluralized_details() {
+        // Given
+        let rowID = Int64(0)
+        let subs: ProductSubscription = createFakeSubscription(periodInterval: "3")
+        let productTypeKey = "subscription"
+
+        let product = Product.fake().copy(productID: 12,
+                                          name: "A subscription product",
+                                          productTypeKey: productTypeKey,
+                                          subscription: subs)
+
+        // When
+        let viewModel = ProductRowViewModel(id: rowID, product: product, productSubscriptionDetails: subs)
+
+        // Then
+        XCTAssertEqual(viewModel.subscriptionBillingDetailsLabel, "$5.00 / 3 months")
+    }
+
+    func test_subscriptionConditionsLabel_when_has_signup_fees_and_trial_period_then_returns_expected_details() {
+        // Given
+        let rowID = Int64(0)
+        let subs: ProductSubscription = createFakeSubscription(periodInterval: "3")
+        let productTypeKey = "subscription"
+
+        let product = Product.fake().copy(productID: 12,
+                                          name: "A subscription product",
+                                          productTypeKey: productTypeKey,
+                                          subscription: subs)
+
+        // When
+        let viewModel = ProductRowViewModel(id: rowID, product: product, productSubscriptionDetails: subs)
+
+        // Then
+        XCTAssertEqual(viewModel.subscriptionConditionsLabel, "$0.60 signup · 1 week free")
+    }
+
+    func test_subscriptionConditionsLabel_when_has_no_signup_fees_then_returns_expected_details() {
+        // Given
+        let rowID = Int64(0)
+        let subs: ProductSubscription = createFakeSubscription(periodInterval: "3", signUpFee: nil)
+        let productTypeKey = "subscription"
+
+        let product = Product.fake().copy(productID: 12,
+                                          name: "A subscription product",
+                                          productTypeKey: productTypeKey,
+                                          subscription: subs)
+
+        // When
+        let viewModel = ProductRowViewModel(id: rowID, product: product, productSubscriptionDetails: subs)
+
+        // Then
+        XCTAssertEqual(viewModel.subscriptionConditionsLabel, "1 week free")
+    }
+
+    func test_subscriptionConditionsLabel_when_has_no_free_trial_then_returns_expected_details() {
+        // Given
+        let rowID = Int64(0)
+        let subs: ProductSubscription = createFakeSubscription(periodInterval: "3", trialLength: nil, trialPeriod: nil)
```

Comment:
Nit: similar comment as above about explicit value for the $0.6 signup fee.

Reply:
Yep! Updated tests on 48f82173d4a186055f9a05467d3ec6b13a630ff9